src/test/scala/com/elevator/ElevatorSystemSpec.scala

```diff
@@ -100,7 +103,7 @@ class ElevatorSystemSpec(implicit ee: ExecutionEnv) extends Specification with R
       (for {
         system <- ElevatorSystem(elevators)
         _ <- system.request(request).fork
-        _ <- system.run(1.millis).fork *> IO.sleep(300.millis)
+        _ <- system.run(millis(1)).fork *> sleep(millis(300))
```

Comment:
we can use 
``` scala
 import scalaz.zio.duration._
300.millis
```
but there would be conflict with the time in some test case using `upTo` 
You can leave it like this though